### DISCUSSION TITLE
Getting pinned tweet

### DISCUSSION THREAD

**Discussion by tejasmhos** [tweepy/tweepy#1178] [Questions] [answered]
There is no mention on how to access the pinned tweet (if any) for the given account.

**Comment by Harmon758** [comment, marked as answer]
This is not something that is accessible through Twitter's API.
See https://twittercommunity.com/t/access-a-pinned-tweet/93721 and https://twittercommunity.com/t/access-pinned-tweet-programatically/87440.
